src/pages-helpers/curriculum/docx/builder/2_tableOfContents.ts

```diff
@@ -53,10 +53,13 @@ export default async function generate(
       }
 
       const anchorId = `section_year_${item.type}-${item.year}`;
+      const text = item.isSwimming
+        ? "Swimming and water safety units (all years)"
```

Comment:
Why are we hard coding `"all years"` here can we not get that from data?

Reply:
`getYearGroupTitle` should do the work required here it should get `Swimming and water safety units` from `groupAs`